mcp-server/src/__tests__/qdrantHandler.integration.test.ts

```
@@ -0,0 +1,115 @@
+  beforeEach(() => {
+    process.env = { ...originalEnv };
+  });
+
+  afterEach(() => {
+    process.env = originalEnv;
+  });
+
+  describe("createEmbedding error handling", () => {
+    it("should throw error when NOMIC_API_KEY is missing", async () => {
+      process.env.EMBEDDING_PROVIDER = "nomic";
+      delete process.env.NOMIC_API_KEY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)